Web/src/AuthPage.js

```diff
@@ -50,12 +49,10 @@ export default function AuthPage() {
                   </button>
                 </div>
                 <div className="form-group">
-                <FacebookLogin
-                  appId="1088597931155576"
-                  autoLoad={true}
-                  fields="name,email"
-                  cssClass="facebook"
-                  icon="fa-facebook" />
+                  <button className="facebook">
```

Comment:
Create a component instead of directly make it here, as it will be used on differents screens.

The syntax is basically the same:
`function FacebookButton() {
        return (
               <button/> // Copy/Paste the code
        )
}`